Source/UI/NewUI/BillingForm/View/Field/BillingFormTextField.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+final class BillingFormTextField: UITextField {
+    var type: BillingFormCellType
```

Comment:
Should this be a let, or could we remove the parameter from init? If this is modifiable but also required at init, its a little inconsistent?